### PR TITLE
[MIG][12.0] website_legal_page

### PR DESCRIPTION
**Pull request by eLBati**

### CODE REVIEW

File: website_legal_page/views/website_legal.xml [hunk 1]
```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- Add legal advice after copyright and company name -->
+    <template id="legal_advice_link" inherit_id="website.layout_footer_copyright" name="Legal advice Link">
+        <xpath expr="//footer//div/span[@t-field='res_company.name']" position="after">
+            <span>-
```
Comment (simahawk) [inline website_legal_page/views/website_legal.xml]: maybe worth to add a class to ease styling here?

File: website_legal_page/views/website_privacy.xml [hunk 2]
```diff
@@ -0,0 +1,246 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- Add privacy policy after copyright and company name -->
+    <template id="privacy_policy_link"
+              inherit_id="website.layout_footer_copyright"
+              name="Privacy policy Link">
+        <xpath expr="//footer//div/span[@t-field='res_company.name']" position="after">
+            <span>-
```
Comment (simahawk) [inline website_legal_page/views/website_privacy.xml]: same

File: website_legal_page/views/website_terms.xml [hunk 3]
```diff
@@ -0,0 +1,256 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- Add terms of use after copyright and company name -->
+    <template id="terms_of_use_link"
+              inherit_id="website.layout_footer_copyright"
+              name="Terms of use Link">
+        <xpath expr="//footer//div/span[@t-field='res_company.name']" position="after">
+            <span>-
```
Comment (simahawk) [inline website_legal_page/views/website_terms.xml]: same

**Review by simahawk — APPROVED** [review]
LG, just one small improvement proposal ;)

**Comment by eLBati** [comment]
@simahawk thanks, done

File: website_legal_page/views/reusable_templates.xml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- Base templates for submodules -->
+    <template id="acceptance_full">
+        I accept the <a href="/legal/advice">legal advice</a>, the
```
Comment (simahawk) [inline website_legal_page/views/reusable_templates.xml]: @eLBati sorry, don't know how but I lost my comment here... same story: can you wrap it (maybe in a <span />) w/ proper CSS class?

**Review by mpanarin — APPROVED** [review]

File: website_legal_page/README.rst [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/186/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module was written to provide common legal pages needed in any website:
+
+* **Legal advice**: Website proprietary and disclaimers
+* **Terms of use**: Which services website offers and legal restrictions applied to them
```
Comment (rafaelbn) [inline website_legal_page/README.rst:31]: @eLBati @simahawk should we reuse the Odoo terms?

https://github.com/odoo/odoo/blob/master/addons/website_sale/views/templates.xml#L1656

When this module was done in v8 Odoo doesn't have terms but Odoo has from v10

What do you think?

Thanks

File: website_legal_page/views/reusable_templates.xml (hunk 4 above)
Comment (eLBati) [reply website_legal_page/views/reusable_templates.xml]: Done

**Comment by eLBati** [comment]
@rafaelbn I propose an 'auto_install' module: website_sale_legal_page. See https://github.com/OCA/website/pull/520

**Comment by rafaelbn** [comment]
@eLBati perfect! Thanks! Please check travis :red_circle: to merge this one! :+1:

File: website_legal_page/readme/CREDITS.rst [hunk 6]
```diff
@@ -0,0 +1,15 @@
+Templates
+~~~~~~~~~
+
+Templates are based on legal templates publicy provided by [termsfeed.com](https://termsfeed.com):
+
+* [Privacy policy](https://media.termsfeed.com/pdf/privacy-policy-template.pdf)
+* [Terms of use](https://media.termsfeed.com/pdf/terms-of-use-template.pdf)
```
Comment (yajo) [inline website_legal_page/readme/CREDITS.rst]: This is a RST file, but you're using Markdown syntax.

File: website_legal_page/readme/CREDITS.rst [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+
+Icon
+~~~~
+
+Icon based on ```johnny-automatic-scales-of-justice.svg``` from
```
Comment (yajo) [inline website_legal_page/readme/CREDITS.rst]: Only 2 backticks

**Review by yajo — CHANGES_REQUESTED** [review]
Looks good, just a small glitch.

**Comment by eLBati** [comment]
@Yajo thanks, done

**Review by yajo — APPROVED** [review]

**Comment by yajo** [comment]
Neat! Can you please squash the 2 last commits to merge? Thanks.

**Comment by eLBati** [comment]
@Yajo I made 2 commits on purpose, because the second one does not properly contain "porting" changes. Do you think it is better to have 1 anyway?

**Comment by yajo** [comment]
In other occasions I would agree with you, but this time IMHO it's better just one. :thinking: 

The reason is simple: in the 1st commit you are creating new `.rst` files that are buggy. I think it doesn't make much sense to introduce a bug on purpose in a brand new file, and then fix it separately later in the same PR.

Do you agree?

**Comment by eLBati** [comment]
@Yajo right, I forgot about README changes. Squashed

**Pull request merged**